MySQL tar ball includes mysql_plugin.1, but mariadb tar ball doesn't.
Since mysql_plugin.1 is added into RPM by hand, we have to add it to the
CMakeLists.txt too.

diff -up mariadb-5.5.28a/man/CMakeLists.txt.mysql_plugin mariadb-5.5.28a/man/CMakeLists.txt
--- mariadb-5.5.28a/man/CMakeLists.txt.mysql_plugin	2013-01-02 11:48:23.845499524 +0100
+++ mariadb-5.5.28a/man/CMakeLists.txt	2013-01-02 11:48:49.967514226 +0100
@@ -21,7 +21,7 @@ SET(MAN1_SERVER innochecksum.1 my_print_
 		mysql_tzinfo_to_sql.1 mysql_upgrade.1 mysql_zap.1
 		mysqld_multi.1 mysqld_safe.1 mysqldumpslow.1 mysqlhotcopy.1
 		mysqlman.1 mysqltest.1 perror.1 replace.1 resolve_stack_dump.1
-		resolveip.1 mysqlbug.1)
+		resolveip.1 mysqlbug.1 mysql_plugin.1)
 SET(MAN8_SERVER mysqld.8 mysqlmanager.8)
 SET(MAN1_CLIENT msql2mysql.1 mysql.1 mysql_find_rows.1 mysql_waitpid.1
 		mysqlaccess.1 mysqladmin.1 mysqlbinlog.1 mysqlcheck.1
